### ISSUE TITLE
Warning after switch to Coq8.8: Focus command deprecated

### ISSUE THREAD

**Issue by benediktahrens** [UniMath/UniMath#966]
E.g.
```
File "./UniMath/CategoryTheory/DisplayedCats/Equivalences.v", line 574, characters 44-51:
Warning: The Focus command is deprecated; use '2: {' instead
[deprecated-focus,deprecated]
```

**Comment by benediktahrens** [comment]
Also the related warning
```
File "./UniMath/CategoryTheory/DisplayedCats/Codomain.v", line 116, characters 4-11:
Warning: The Unfocus command is deprecated [deprecated-unfocus,deprecated]
```

**Comment by benediktahrens** [comment]
Solved in #974
